### PR TITLE
Automatic build of woff and css

### LINKED ISSUE #11: Save woff and css with action
Can we add something like this to test-build.yml (similar to https://github.com/abapGit/abapGit/blob/master/.github/workflows/build.yml)?

```yml
    - name: Upload font and css 
      if: always()
      uses: actions/upload-artifact@main
      with:
        name: ag-icons.woff
        path: ./build/ag-icons.woff
        name: ag-icons.css
        path: ./build/ag-icons.css
        retention-days: 7
```

This way we won't need to setup our own nodejs etc. 

PS: Not sure if my syntax is correct.

### PR DESCRIPTION
**Pull request by mbtools**
- Updated build action
- Updated readme

Works like a charm. No local build necessary anymore. 

Closes #11

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - uses: actions/setup-node@v1
+    - uses: actions/checkout@v2
+    - name: Run build
+      run: |
+        npm install
+        npm run build
+        test -f build/ag-icons.woff
+        test -f build/ag-icons.css
+    - name: Upload font and css
+      if: always()
```
Comment (sbcgua) [inline .github/workflows/build.yml:24]: Hmm, why always ?

File: README.md [hunk 2]
```diff
@@ -2,26 +2,29 @@
...
+Icons in abapGit are displayed using a specially build font. As a default, abapGit uses icons from the free distribution of [Font Awesome](https://fontawesome.com/). When adding other icons mind the license matters please. Glyphs must be licensed for free usage (MIT, CC, SIL OFL).
+
+Note: The font is not limited to icons. For example, it include the abapGit logo (Git icon + abapGit text).
+
 ## Build
 
-You need nodejs to be installed (version 10 at the time of package creation)
+On every commit and pull request, a Github action will build the abapGit icon font and corresponding CSS files. The files are available as an artifact in the action and you can download them from there.
 
-```
```
Comment (sbcgua) [inline README.md:9]: I would leave the manual procedure as a subsection of docs. This might be useful in case of issues or further devs

**Review by sbcgua — CHANGES_REQUESTED** [review]
looks nice, but pls have a look at comments. Also I'd suggest separating commit for server-solid

File: README.md (hunk 2 above)
Comment (mbtools) [reply README.md:9]: +1

**Comment by sbcgua** [comment]
OK, let's merge and try :)

**Pull request merged**